docs/_data/navigation.yml

```diff
@@ -38,3 +38,13 @@ footer-resources:
   - title: FAQ
     url: https://docs.secretless.io/Latest/en/Content/FAQ/faq.htm
     target: target="_blank" rel="noopener noreferrer"
+
+tutorial-tiles:
```

Comment:
shouldn't this just be referencing the k8s tutorial? what if we have more tutorials down the line?

Reply:
We could create 'Collections' but that would just be the same thing. For more tutorials, we would add another set of navigation like this one. @typaulhus do you know a work around?

Reply:
addressed in issue created #680

Reply:
changed to `tutorial-k8s:`